crystal/lucky/config.yaml

```diff
@@ -1,3 +1,5 @@
 framework:
   website: luckyframework.org
+  env:
```

Comment:
you have to set an environment variable for the running process ?

Reply:
I'm not sure I understand. I set LUCKY_ENV to production since that is typical for a Lucky application. It probably won't make a difference since this is a pretty bare app so I can remove it if you want

Reply:
Ah, like `KEMAL_ENV`. It's then an environment variable. Let me update the codebase